homeworks/olha.tymchyshyn_olyatmchshn/PokemonProject/pokemons.html

```diff
@@ -26,12 +26,40 @@
                         <li class="nav-menu-item">
                             <a class="nav-menu-item-link" href="./index.html">About</a>
                         </li>
+                        <li class="nav-menu-item">
+                            <a class="nav-menu-item-link" href="./login.html">Login</a>
+                        </li>
                     </ul>
                 </nav>
             </div>
         </header>
 
         <main>
+            <form action="" class="filter">
+                <div class="type-container">
+                    <select name="type">
+                        <option value="option_1" selected disabled>Type</option>
+                        <option value="option_2">Option #1</option>
+                        <option value="option_3">Option #2</option>
+                    </select>
+                </div>
+                <div class="radio-container">
+                    <input type="radio" name="newOrFav" value="new">
+                    <span class="radio-span">Only new </span>
```

Comment:
Hey, @olyatmchshn, please use the semantically right element `label` for this description and connect it to inputs to fulfill the task requirements
> По кліку на підпис інпута, він має отримувати фокус(поле пошуку), або змінювати стан (чекбокс, радіокнопка).